packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-expandable-answer.pcss

```diff
@@ -0,0 +1,36 @@
+@import '../../global/global.pcss';
+@import '../../global/mixins.pcss';
+
+:host {
+  atomic-smart-snippet-answer {
+    @mixin set-font-size var(--atomic-text-lg);
+
+    display: block;
+    overflow: hidden;
+    height: var(--collapsed-size);
+    transition: height ease-in-out 0.25s;
+
+    --gradient-start: var(
+      --atomic-smart-snippet-gradient-start,
```

Comment:
If at any point a client decides they want to personalize where the gradient starts, it can be done by changing `--atomic-smart-snippet-gradient-start`.

Reply:
Did you document it like
```
/**
 * @prop --atomic-smart-snippet-gradient-start: ......
 */
````